system/Session/Handlers/FileHandler.php

```diff
@@ -67,9 +67,9 @@ public function __construct(AppConfig $config, string $ipAddress)
     {
         parent::__construct($config, $ipAddress);
 
-        if (! empty($config->sessionSavePath)) {
-            $this->savePath = rtrim($config->sessionSavePath, '/\\');
-            ini_set('session.save_path', $config->sessionSavePath);
+        if (! empty($this->savePath)) {
```

Comment:
Why did this change?

Reply:
Because `$this->savePath` is set in the parent class.
See https://github.com/codeigniter4/CodeIgniter4/pull/6989/files#diff-d0ff4c461b6990417b08b47e7967ea04338dd557b3a04155896940c117a3a37fR122